extensions/labeled-response.md

```diff
@@ -77,6 +72,12 @@ When a client sends a message to itself, the server MUST NOT include the label t
 
 The tag value is chosen by the client and MUST be treated as an opaque identifier. The client SHOULD NOT reuse a tag value until it has received a complete response for that value from the server.
 
+### The `ACK` command
+
+Servers MUST respond with a labeled `ACK` message when a client sends a labeled command that normally produces no response. It takes no additional parameters and is defined as follows
```

Comment:
> Servers SHOULD respond with a labeled `ACK` message (and SHOULD NOT send an empty `BATCH`) when a client sends a labeled command that normally produces no response. Clients SHOULD handle an empty `BATCH` the same as it would handle `ACK`. `ACK` takes no additional parameters and is defined as follows

Reply:
Eh, I think this particular language is a bit verbose but it feels right to ensure clients can process empty labeled-response batches. Wording like this feels like it should work and express the same thing (adding only the _"rather than an empty batch"_ text), it makes it clear that one or the other is required (if that needs to be made more clear the suggestion earlier in the doc should) and that servers should default to sending the `ACK`

> Servers SHOULD respond with a labeled `ACK` message rather than an empty batch when a client sends a labeled command that normally produces no response. It takes no additional parameters and is defined as follows

-----

I tend to associate 'command' with something sent from the client to the server so might make sense to change the name of this section from  "The `ACK` command" to "The `ACK` message", but eh that's nitpicking and maybe not a widely-held assumption.

Reply:
works for me

Reply:
lgtm, thanks!